deps.ts

```diff
@@ -0,0 +1,15 @@
+export * from "https://raw.githubusercontent.com/lucacasonato/fresh/04a32f34987fbd59ce646967f216b2cca072a58f/runtime.ts";
```

Comment:
Do you need this specific version? Has it not been tagged yet?

Reply:
fresh isnt on /x/